frameworks/zsh4humans.zsh

```diff
@@ -0,0 +1,24 @@
+(
+
+emulate -L zsh -o err_return -o no_unset -o no_aliases
+cd -- $1
+# download zsh4humans v5
+curl -fsSLO https://github.com/romkatv/zsh4humans/archive/refs/heads/v5.tar.gz
+# copy over .zshenv and .zshrc
+tar -xzf v5.tar.gz
+cp zsh4humans-5/{.zshenv,.zshrc} ./
+rm -r v5.tar.gz zsh4humans-5
+# create a config for powerlevel10k
+>.p10k.zsh <<<$'POWERLEVEL9K_MODE=ascii\nPOWERLEVEL9K_DISABLE_CONFIGURATION_WIZARD=true'
```

Comment:
Can we maybe install zimfw/git, zimfw/termtitle and zimfw/utility, and not install fzf or homebrew-not-found for feature parity?

Reply:
> Can we maybe install zimfw/git

Sure. Done.

> zimfw/termtitle and zimfw/utility

zsh4humans has similar things builtin.

>  not install fzf or homebrew-not-found for feature parity?

This cannot be done.

Frameworks under benchmark are allowed to have extra features that zimfw doesn't have, right?

Reply:
We try to make the features be as close as possible to the Zim default installation. As this is the closest you consider we should get, then it's good!